src/main/java/step5/domain/RacingGame.java

```diff
@@ -0,0 +1,52 @@
+package step5.domain;
+
+/*
+ * Racing Game
+ * ver. 1.0
+ * 2020.05.27
+ * Copyright ...
+ */
+public class RacingGame {
+
+    private static final String ILLEGAL_INPUT_ERROR = "Inputs can't be negative number.";
+    private static final String RANKED_STRING_DELIMITER = ",";
+
+    private Cars cars;
+
+    private int playTimes;
+
+    private RacingGame() {
+    }
+
+    public RacingGame(String[] carsNames, int playTimes) {
+
+        int carsCount = carsNames.length;
+        if (carsCount <= 0 || playTimes < 0) throw new IllegalArgumentException(ILLEGAL_INPUT_ERROR);
+
+        /* set new car */
+        this.cars = Cars.of(carsNames, 0, new CarMoveRandomStrategy());
+
+        /* set play times. */
+        this.playTimes = playTimes;
+
+    }
+
+
+    public Cars play() {
+
+        cars.move();
+
+        return cars;
```

Comment:
방어적 복사를 하는게 좋을 것 같습니다 :) 외부에서 반환된 Cars 객체를 조작할 염려가 있을 거 같아요
Cars 객체에 자기 자신을 받는 정적 팩토리 메소드를 하나 만들고
```
public static Cars newInstance(Cars cars) {
        Cars newCars = new Cars();
        newCars.carsList.addAll(cars.carsList);
        return newCars;
    }
```
(위 코드 역시 참고만 해주세요^^)
```suggestion
        return Cars.newInstance(cars);
```
를 사용하면 좋을 것 같습니다 :) 참고해주세요!